src/diamond/convertor.py

```diff
@@ -32,47 +51,28 @@ def set(self, value, unit=None):
     def get(self, unit=None):
         return self.do(unit=unit)
 
+    def _object_mapper(self, unit, value, mapping_list, object_type):
+        for counter, units in enumerate(mapping_list):
+            if unit in units:
+                # returning self.type_of_unit
+                return self.convertb(value, object_type, counter)
+        return "None"
```

Comment:
I don't see how there's a difference between `if obj == "None"` and `if obj is None` in L67 &c. :confused: